src/data/policies.ts

```diff
@@ -46,7 +46,7 @@ export type UpdatePolicyRequestBody = {
 
 export const policies = writable<Policy[]>([])
 export const selectedPolicy = derived([policies, selectedPolicyId], ([policies, selectedPolicyId]) => {
```

Comment:
when other stores are referenced in a derived store they should be prepended with a $
```suggestion
export const selectedPolicy = derived([policies, selectedPolicyId], ([$policies, $selectedPolicyId]) => {
```